packages/energy-api-influxdb/src/reads/dto/measurement.dto.ts

```diff
@@ -1,10 +1,12 @@
 import { Unit } from '../unit.enum';
 import { IsEnum, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
 import { ApiProperty } from '@nestjs/swagger';
 import { ReadDTO } from './';
 
 export class MeasurementDTO {
   @ValidateNested()
+  @Type(() => ReadDTO)
```

Comment:
This was missing for the validation to be working:
https://github.com/typestack/class-validator#validating-nested-objects
https://github.com/typestack/class-transformer#working-with-nested-objects